google/cloud/firestore_v1/base_client.py

```diff
@@ -177,15 +177,20 @@ def _emulator_channel(self, transport):
         # TODO: Implement a special credentials type for emulator and use
         # "transport.create_channel" to create gRPC channels once google-auth
         # extends it's allowed credentials types.
+        # if "GrpcAsyncIOTransport" in str(transport.__name__):
+        #     return grpc.aio.secure_channel(
+        #         self._emulator_host, self._local_composite_credentials()
+        #     )
+        # else:
+        #     return grpc.secure_channel(
+        #         self._emulator_host, self._local_composite_credentials()
+        #     )
```

Comment:
Do we really want to leave this commented-out bit in place?

Reply:
no ;) but this is mostly here to spark conversation. I am not sure we can even make this change without regressing something.